### PR TITLE
Clarify page activity calculation

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?
Clarifies how page activity tracking works.

### Motivation
Internal feedback after customer interaction: [DOCS-5163](https://datadoghq.atlassian.net/browse/DOCS-5163).

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5163]: https://datadoghq.atlassian.net/browse/DOCS-5163?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 1]
```diff
@@ -86,14 +86,19 @@ To account for modern web applications, loading time watches for network request
...
 
 ### How page activity is calculated
 
-The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page is deemed to have activity by looking at network requests and DOM mutations. The page activity ends when there are no ongoing requests and no DOM mutation for more than 100ms. The page is determined to have no activity if no requests or DOM mutation occurred in 100ms.
+The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page activity tracker is calculated through the following process:
+
+- Check ongoing `xhr` or `fetch` requests
+- Check if resources (js, css, and such) ended in a 100ms sliding window
+
+ The page activity ends when there are no ongoing requests and no DOM mutation for more than 100ms. The page is determined to have no activity if no requests or DOM mutation occurred in 100ms.
```
Comment (bcaudan) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:101]: **💬 suggestion:** ‏
```suggestion
The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page is considered to have activity when:

- `xhr` or `fetch` requests are in progress
- Performance resource timing entries are emitted by the browser (loading end of js, css, and such)
- DOM mutations are emitted by the browser

The page activity is considered ended when the page don't have activity for 100ms.

**Note**: only activity happening after the SDK init is taken into account.
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 2]
```diff
@@ -86,14 +86,19 @@ To account for modern web applications, loading time watches for network request
 
 - **Initial Load**: Loading Time is equal to _whichever is longer_:
 
-  - The difference between `navigationStart` and `loadEventEnd`.
-  - Or the difference between `navigationStart` and the first time the page has no activity. Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
+  - The difference between `navigationStart` and `loadEventEnd`, or
+  - The difference between `navigationStart` and the first time the page has no activity for 100ms (with activity defined as ongoing network requests or a DOM mutation). Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
```
[the remark this answers is not in the document]
Comment (rtrieu) [reply content/en/real_user_monitoring/browser/monitoring_page_performance.md]: Will change this back. Thanks!

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 3]
```diff
@@ -86,14 +86,19 @@ To account for modern web applications, loading time watches for network request
...
 
 - **SPA Route Change**: Loading Time is equal to the difference between the URL change and the first time the page has no activity. Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
 
 ### How page activity is calculated
 
-The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page is deemed to have activity by looking at network requests and DOM mutations. The page activity ends when there are no ongoing requests and no DOM mutation for more than 100ms. The page is determined to have no activity if no requests or DOM mutation occurred in 100ms.
+The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page activity tracker is calculated through the following process:
+
+- Check ongoing `xhr` or `fetch` requests
+- Check if resources (js, css, and such) ended in a 100ms sliding window
```
Comment (rtrieu) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md]:
```suggestion
- Performance resource timing entries are emitted by the browser (loading end of js, css, and such)
- DOM mutations are emitted by the browser
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md (hunk 1 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/browser/monitoring_page_performance.md]: Thank you, @bcaudan !

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 4]
```diff
@@ -86,14 +86,22 @@ To account for modern web applications, loading time watches for network request
...
 
 - **SPA Route Change**: Loading Time is equal to the difference between the URL change and the first time the page has no activity. Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
 
 ### How page activity is calculated
 
-The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page is deemed to have activity by looking at network requests and DOM mutations. The page activity ends when there are no ongoing requests and no DOM mutation for more than 100ms. The page is determined to have no activity if no requests or DOM mutation occurred in 100ms.
+The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page is considered to have activity when:
+
+- `xhr` or `fetch` requests are in progress
+- Performance resource timing entries are emitted by the browser (loading end of js, css, and such)
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:99]:
```suggestion
- The browser emits performance resource timing entries (loading end of JS, CSS, etc.).
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 5]
```diff
@@ -86,14 +86,22 @@ To account for modern web applications, loading time watches for network request
...
+- DOM mutations are emitted by the browser
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:100]:
```suggestion
- The browser emits DOM mutations.
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 6]
```diff
@@ -86,14 +86,22 @@ To account for modern web applications, loading time watches for network request
...
+  - The difference between `navigationStart` and the first time the page has no activity. Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
 
 - **SPA Route Change**: Loading Time is equal to the difference between the URL change and the first time the page has no activity. Read [How page activity is calculated](#how-page-activity-is-calculated) for details.
 
 ### How page activity is calculated
 
-The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page is deemed to have activity by looking at network requests and DOM mutations. The page activity ends when there are no ongoing requests and no DOM mutation for more than 100ms. The page is determined to have no activity if no requests or DOM mutation occurred in 100ms.
+The RUM Browser SDK tracks the page activity to estimate the time until the interface is stable again. The page is considered to have activity when:
+
+- `xhr` or `fetch` requests are in progress
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:98]:
```suggestion
- `xhr` or `fetch` requests are in progress.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
small nits

**Review by rtrieu — COMMENTED** [review]
Accepted all suggestions. Thanks, @cswatt !

**Review by cswatt — APPROVED** [review]

**Pull request merged**